scaladoc/src/dotty/tools/scaladoc/ScaladocSettings.scala

```diff
@@ -94,3 +95,12 @@ class ScaladocSettings extends SettingGroup with AllScalaSettings:
 
   val YdocumentSyntheticTypes: Setting[Boolean] =
     BooleanSetting("-Ydocument-synthetic-types", "Documents intrinsic types e. g. Any, Nothing. Setting is useful only for stdlib", false)
+
+  val snippetCompiler: Setting[List[String]] =
+    MultiStringSetting("-snippet-compiler", "snippet-compiler", snippets.SnippetCompilerArgs.usage)
+
+  val snippetCompilerDebug: Setting[Boolean] =
+    BooleanSetting("-snippet-compiler-debug", snippets.SnippetCompilerArgs.debugUsage, false)
```

Comment:
Flags which are for debugging purpose only and not intended for others to use should start with -Y